src/v2/connectors/redisGraph.js

```diff
@@ -304,7 +304,7 @@ export default class RedisGraphConnector {
   async runApplicationsQuery() {
     const { withClause, whereClause } = await this.createWhereClause([], ['app']);
     const matchClause = `MATCH ${APPLICATION_MATCH} ${whereClause}`;
-    const returnClause = 'RETURN DISTINCT app._uid, app.name, app.namespace, app.created, app.dashboard, app.selfLink, app.label ORDER BY app.name, app.namespace ASC';
```

Comment:
@KevinFCormier Can you confirm that removing `selfLink` from these 4 queries won't cause problems in the App UI?  I'm pretty sure you have removed selfLink, but wanted to double check.